pkgs/development/libraries/libopenblt/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchFromGitHub, cmake, libusb1 }:
+
+stdenv.mkDerivation rec {
+
+  pname = "libopenblt";
+  version = "011500";
+
+  src = fetchFromGitHub {
+    owner = "feaser";
+    repo = "openblt";
+    rev = "openblt_v${version}";
+    sparseCheckout = [ "Host/Source/" ];
+    sha256 = "sha256-cWdc6Imd6IBqYQRNEafp3bDDQLgdpE90fxSkY+vzhAM=";
+  };
+
+  preConfigure = "cd ./Host/Source/LibOpenBLT/";
+
+  installPhase = ''
+    install -D $NIX_BUILD_TOP/source/Host/libopenblt.so $out/lib/libopenblt.so
+    install -D $NIX_BUILD_TOP/source/Host/Source/LibOpenBLT/openblt.h $out/include/openblt.h
```

We can rely on pwd here

```suggestion
    install -D Host/libopenblt.so $out/lib/libopenblt.so
    install -D Host/Source/LibOpenBLT/openblt.h $out/include/openblt.h
```

---

tried, doesn't work

---

During installphase I'm here `/build/source/Host/Source/LibOpenBLT/build` but I need to take my file from here `/build/source/Host/`. I would like to be in the `/build/source` folder so I can correctly type out the path location and don't use something not understandable like `../../../libopenblt.so`

---

Check the solution on the last commit